docs/source/data/fetching-rest.mdx

```diff
@@ -210,74 +261,92 @@ For a simple example, suppose our REST endpoint responded to the following URLs:
 
 A "malicious" client could provide an `:id` of `1/characters` to target the delete `characters` endpoint when it was the singular `movie` endpoint that we were trying to delete. URI encoding prevents this kind of injection by transforming the `/` into `%2F`. This can then be correctly decoded and interpreted by the server and won't be treated as a path segment.
 
-#### Method parameters
+### Method parameters
 
 For all HTTP convenience methods, the **first parameter** is the relative path of the endpoint you're sending the request to (e.g., `movies`).
 
 The **second parameter** depends on the HTTP method:
 
-- For HTTP methods with a request body (`post`, `put`, `patch`), the second parameter _is_ the request body.
+- For HTTP methods with a request body (`post`, `put`, `patch`), the second parameter _is_ the request body (e.g., `{ body: { id: movie.id } }`).
```

Comment:
The second parameter is actually an object which contains the request body (which you've correctly illustrated in your example). It also includes `method` and potentially `cacheOptions`.

Reply:
Oh, thank you, I forgot to correct the sentence before the example 🤦‍♀️